site/en/docs/extensions/mv3/manifest/input_component/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Manifest - input_component"
+date: 2022-10-28
+updated: 
+description: Reference documentation for the input_component property of manifest.json.
+---
+
+An optional Manifest key enabling the use of the  [input.ime API](/docs/extensions/reference/input_ime/) (Input Method Editor) for use with ChromeOS. This allows your extension to handle keystrokes, set the composition, and open assistive windows.   
+Including the key in the extension manifest is done under Permissions.
```

Comment:
Either add an extra newline to make these two distinct paragraphs or remove the newline to make them a single paragraph.

```suggestion
An optional Manifest key enabling the use of the  [input.ime API](/docs/extensions/reference/input_ime/) (Input Method Editor) for use with ChromeOS. This allows your extension to handle keystrokes, set the composition, and open assistive windows.   

Including the key in the extension manifest is done under Permissions.
```

Reply:
This should be one paragraph.